.github/workflows/ci.yml

```diff
@@ -91,6 +91,13 @@ jobs:
           - should-release: false
             os: macos-latest
     steps:
+      - uses: actions/checkout@v4
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
       - name: Build wheelhouse and perform smoke test
         uses: ansys/actions/build-wheelhouse@v6
```

Comment:
Python is already installed within this step. I'm closing the issue.

Reply:
Sure.